File: R/tests/testthat/test_bart_stl.R
context("Test bart_stl")
library(hBayesDM)

test_that("Test bart_stl", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_stl(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_bart_stl.py
import pytest

from hbayesdm.models import bart_stl


def test_bart_stl():
    _ = bart_stl(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
